app/views/categories/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="category-details-container">
+  <div class="header">
+    <%= link_to '←', categories_path, class: 'back-link' %>
+    <h1 class="category-name"><%= @category.name %></h1>
+  </div>
+
+  <div class="transactions-section">
+  
+    <h2>TRANSACTIONS</h2>
+
+    <div class="category-info">
+      <div class="category-info-header">
+        <picture>
+          <img src="<%= @category.icon %>" alt="icon">
+        </picture>
+        <div>
+          <h3><%= @category.name %></h3>
+          <p>📅<%= @category.created_at.strftime("%d %b %Y at %I:%M %p") %></p>
+        </div>
+        <p><%= number_to_currency(@total_amount, precision: 2) %></p>
```

Comment:
- [optional] It would be nice if you can add a vertical scroll bar to this page for good user experience. ℹ️ 
 
<img width="307" alt="Untitled" src="https://github.com/ahmidou09/budget_manager/assets/103386183/cec06cef-263a-4b39-adf9-83b677b993b3">